jenkins/root-build.cmake

```diff
@@ -757,6 +758,16 @@ if("$ENV{BUILDOPTS}" STREQUAL "cxxmodules" OR
   unset(CTEST_CHECKOUT_COMMAND)
 endif()
 
+#---Inject environment variables for fine granular control over expected test failures----------------
+set(NODENAMES_IGNORE_NUMBA # List of nodenames on which we expect numba tests to fail
+    macphsft19.dyndns.cern.ch # numba cannot be installed
```

Comment:
I'd prefer to do that per config instead of relying on node names: VMs might get replaced. I.e. please set that var in a dedicated `if("${LABEL}" MATCHES "ubuntu18|debian")` (or whatever the condition is for Linux), and in the `GET_ALL_SUPPORTED_MODULES_APPLE` function.

Reply:
i agree, makes more sense. if you agree that we just disable generically on mac os, the PR should be good to go. on mac, we fail for py2 because it's the system one but we are fine for py3, because it's installed by ourselves (i guess?) and we have pip availabe. because the configuration comes after the selection of the modules, we cannot find out which python is picked up (?) if we can figure out which python we use by the flags provided in this stage, we could add a `AND NOT python3` or something similar to increase the test coverage.